src/components/FeatureChecking/flowTypes.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+export type FeatureOptions = {
```

Comment:
FeatureOptions = Object?
Since thats an object

Reply:
"Object" is disallowed in strict mode, and is broader than this type (lots of things are objects).